jvcl/run/JvAni.pas

```diff
@@ -40,6 +40,9 @@ interface
   {$IFDEF UNITVERSIONING}
   JclUnitVersioning,
   {$ENDIF UNITVERSIONING}
+  {$IFDEF HAS_UNIT_TYPES}
+  System.Types,
```

Comment:
Do not prefix, the namespace support was added later on.
Basically, Types could have been simply added at the end of the next line

Reply:
Ok, good to know, so I changed it.